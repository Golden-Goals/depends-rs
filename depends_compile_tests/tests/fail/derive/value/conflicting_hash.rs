use depends::derives::Value;

#[derive(Value)]
#[depends(unhashable)]
struct Foo<T> {
    bar: Vec<T>,
    #[depends(hash)]
    number: usize,
}

fn main() {}
