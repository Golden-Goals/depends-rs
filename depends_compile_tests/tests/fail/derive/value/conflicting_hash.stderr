error: Attribute specified more than once
 --> tests/fail/derive/value/conflicting_hash.rs:7:15
  |
7 |     #[depends(hash)]
  |               ^^^^
